sections/main-collection-product-grid.liquid

```diff
@@ -136,17 +133,18 @@
                         <input class="field__input"
                           name="{{ filter.max_value.param_name }}"
                           id="Filter-{{ filter.label | escape }}-LTE"
-                          {%- if filter.max_value.value -%}
-                            {%- if uses_comma_decimals -%}
-                              value="{{ filter.max_value.value | money_without_currency | replace: '.', '' | replace: ',', '.' }}"
-                            {%- else -%}
-                              value="{{ filter.max_value.value | money_without_currency | replace: ',', '' }}"
-                            {% endif %}
+                          {%- if filter.max_value.value -%}{%- if uses_comma_decimals -%}value="{{ filter.max_value.value | money_without_currency | replace: '.', '' | replace: ',', '.' }}"{%- else -%}value="{{ filter.max_value.value | money_without_currency | replace: ',', '' }}"{% endif %}
                           {%- endif -%}
                           type="number"
-                          placeholder="{{ filter.range_max | money_without_currency | replace: ',', '' }}"
                           min="0"
-                          max="{{ filter.range_max | money_without_currency | replace: ',', '' }}">
+                          {%- if uses_comma_decimals -%}
+                            placeholder="{{ filter.range_max | money_without_currency | replace: '.', '' | replace: ',', '.' }}"
```

Comment:
We've tried to avoid `{% assign %}` as much as possible in the theme, but could the `filter.range_max`, which is repeated a few times, benefit from using it?

```
{%- if uses_comma_decimals -%}                 
  {%- assign range_max = filter.range_max | money_without_currency | replace: '.', '' | replace: ',', '.' -%}
{%- else -%}
  {%- assign range_max = filter.range_max | money_without_currency | replace: ',', '' -%}
{%- endif -%}

placeholder="{{ range_max }}"
max="{{ range_max }}"                     
```

Reply:
I'd rather avoid it for now.  We're going to revisit this approach anyway with some help from the platform given how fragile it is, so it'll likely be something a lot more manageable that wouldn't justify an assign down the line. See #255 for more details.